### ISSUE TITLE
Collapsable Text Selections

### ISSUE THREAD

**Issue by ventolinmx** [giuspen/cherrytree#2360]
It would be great to have collapsable sections of text inside a node where users could select a piece of text and hide it and expand it with a little [+] button. This helps to read different parts of a note at the same time that are big. For example, suppose I have on my note a title and a general description. Then I have many different sections below but on each I would like to refer constantly to the general description because it has an important definition or something. This way i can make the description visible and navigate through the rest of the contents. Also it would help to hide less relevant, detailed or redundant info and keep only the main sections in view that would help skim through the contents more efficiently.

**Comment by guidomz** [comment]
I concur. Some function similar to html's 
```
<details>
    <summary>Title here</summary>
    Content here
</details>

**Comment by RFImages** [comment]
Yes, Word does this (using headings) and it is extremely useful for very long text bodies. Please implement this - the headings are already there  (h1,h2,h3) make them collapsible please!

**Comment by RFImages** [comment]
I am noting other requests and curious where this might be in the queue

https://github.com/giuspen/cherrytree/issues/2389

https://github.com/giuspen/cherrytree/issues/2026

**Comment by RFImages** [comment]
@giuspen  Perhaps this can be closed because the main thread on this subject is #2389 ?
